README.md

```diff
@@ -58,7 +58,7 @@ releases of the toolkit.
 * Optional, but recommended:
     * Gradle 3.1 or greater, needed for building the GATK. We recommend using the `./gradlew` script which will
       download and use an appropriate gradle version automatically (see examples below).
-    * Python 2.6 or greater (needed for running the `gatk-launch` frontend script)
+    * [THIS SECTION NEEDS TO BE UPDATED WITH CONDA INFO] Python 2.6 or greater (needed for running the `gatk-launch` frontend script)
```

Comment:
Is this happening in this PR or a later one?  Do we need to file an issue?

Reply:
I want to get at least something in here as part of this PR.